features/README.md

```diff
@@ -0,0 +1,131 @@
+# Ably Features
+
+## Introduction
+
+The data and code files in this folder are incubating as a prototype, hopefully the eventually-accepted foundations, of Ably's approach to SDK feature tracking.
+
+These are the pivotal components:
+
+| Component | Purpose |
+| ---- | ------- |
+| [`sdk.yaml`](sdk.yaml) | The canonical feature list that Ably SDKs can support, represented as a tree. |
+| [`sdk-manifests/`](sdk-manifests/) | Temporary location (while we prototype) for manifests, per SDK source code repository, declaring what features that SDK supports. These files will ultimately be moved to be managed as peer-level source code in each SDK's source code repository. |
```

Comment:
Just to make sure I understand the nature of your comment, are you asking for a stronger indication at this stage in the document that the manifest uses the same keys as defined in the canonical features list?

If so, then I can see that this could be expanded to make that clearer, at this Introduction stage. It's covered a little later on in the document (i.e. in Future Direction for the SDK Manifests) but I can see that it could help the reader to have the dots more obviously joined earlier on.

Reply:
Sorry, I missed to answer this one - Yes that was my intention, but I think this would be an improvement rather than a necessity. I will resolve this conversation to get the PR forward